api/job_resources.go

```diff
@@ -0,0 +1,28 @@
+package api
+
+import (
+	"github.com/hashicorp/nomad/nomad/structs"
+)
+
+type JobResources struct {
```

Comment:
Yes, that was one thing I wasn't sure about, as currently a resource in the api package is commented as "Resource encapsulates the required resources of a given task or task group."